casper/src/main/scala/coop/rchain/casper/Estimator.scala

```diff
@@ -70,7 +70,7 @@ object Estimator {
 
     for {
       scoresMap           <- buildScoresMap[F](blockDag)
-      sortedChildrenHash  = sortChildren(IndexedSeq(genesis.blockHash), blockDag.childMap, scoresMap)
+      sortedChildrenHash  = sortChildren(IndexedSeq(genesisBlockHash), blockDag.childMap, scoresMap)
       maybeSortedChildren <- sortedChildrenHash.toList.traverse(BlockStore[F].get)
```

Comment:
The code `BlockStore[F].get` is still needed here, because we need to get the deploy of these block in `ProtoUtil.conflicts`.